### DISCUSSION TITLE
Welcome to mermaid-powerbi Discussions!

### DISCUSSION THREAD

**Discussion by zBritva** [zBritva/mermaid-powerbi#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by andremoraes** [comment]
hello, is possible to have the visual to render data inside a table, cell?  not from the editor ? so I can dynamically have the visual to render data created in a database?

**Reply by zBritva** [reply]
There is Handlebars.js for data binding: https://ilfat-galiev.im/docs/markdown-visual/#data-binding-with-handlebarsjs-template

**Reply by klitmoeller** [reply]
Dear Ilfat,

handlebars.js are great, but unfortunately if i use this:

````mermaid
{{#each table.rows}}
{{ this.test }}
{{/each}}`

the stored data 
"flowchart TD
    Start --> Stop" 

is turned into this (using html codes) which prevents the rendering:
"flowchart TD
    Start --&gt; Stop"

Is there a way to fix that ?

Thank you so much!!

**Reply by zBritva** [reply]
@klitmoeller use triple brackets:

![image](https://github.com/user-attachments/assets/49ff9a68-a4f6-49f1-a57c-0f9615152481)

\```mermaid
{{#each table.rows}}
{{{ this.test }}}
{{/each}}
\```

[Double applies html escaping](https://handlebarsjs.com/guide/#html-escaping)

**Reply by andremoraes** [reply]
I will try this out thanks so much for sharing

**Reply by klitmoeller** [reply]
@zBritva  🙏 very much appreciated! Thank you!

**Comment by CollierTR** [comment]
Hello everyone,

My name is Tristan. I'm a BI Developer by day and a self taught developer by night. I recently started using this plugin in our team's workflow. I'd love to look into ways I can contribute to this community!
